app/controllers/plans_controller.rb

```diff
@@ -168,7 +168,6 @@ def edit
     render('/phases/edit', locals: {
       base_template_org: phase.template.base_org,
       plan: plan, phase: phase, readonly: readonly,
-      question_guidance: plan.guidance_by_question_as_hash,
```

Comment:
Removed this method. Doesn't seem to be used in the code, and the variable set here doesn't seem to be used in the views.

Reply:
thanks for catching that. we must have missed it in the rewrite of the pages